### DISCUSSION TITLE
Is the Luv to uv conversion correct ?

### DISCUSSION THREAD

**Discussion by spicymatt** [colour-science/colour#743] [Q&A] [answered]
Hello,
I am confused by the LUv to uv conversion formula for v that uses 9 * Y / (X + 15 * Y + 3 * Z) when the published formula uses 6*Y/.... (cf https://en.wikipedia.org/wiki/CIE_1960_color_space#Relation_to_CIE_1976_UCS)
Am I missing something ?
Matthieu

**Comment by sobotka** [comment, marked as answer]
https://en.wikipedia.org/wiki/CIELUV

**Comment by spicymatt** [comment]
> https://en.wikipedia.org/wiki/CIELUV

Thanks...and sorry for the confusion. The 1976 definition supercedes the 1960 definition for 'v'. 
I missed that.
